<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestFrame</class>
 <widget class="QFrame" name="TestFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <property name="horizontalSpacing">
    <number>40</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="5" column="0">
    <widget class="MDSpinBox" name="sbFrequency">
     <property name="label" stdset="0">
      <string>Frequency [Khz]</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="MDSpinBox" name="sbTileSize">
     <property name="label" stdset="0">
      <string>Tile size [mm]</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="MDSpinBox" name="sbPointsPerTile">
     <property name="label" stdset="0">
      <string>Points per Tile</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pbStartProcess">
     <property name="text">
      <string>Start process</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="MDSpinBox" name="sbPulses">
     <property name="label" stdset="0">
      <string>Pulses</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="MDLineEdit" name="leFilePath">
     <property name="label" stdset="0">
      <string>Path File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MDLineEdit</class>
   <extends>QLineEdit</extends>
   <header>gui/custom-widgets/MDLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>MDSpinBox</class>
   <extends>QSpinBox</extends>
   <header>gui/custom-widgets/MDSpinBox.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leFilePath</tabstop>
  <tabstop>sbPointsPerTile</tabstop>
  <tabstop>sbTileSize</tabstop>
  <tabstop>sbPulses</tabstop>
  <tabstop>sbFrequency</tabstop>
  <tabstop>pbStartProcess</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
